src/utils/daos.c

```diff
@@ -816,7 +875,9 @@ common_op_parse_hdlr(int argc, char *argv[], struct cmd_args_s *ap)
 	/* Verify pool svc argument. If not provided pass NULL list to libdaos,
 	 * and client will query management service for rank list.
 	 */
-	ARGS_VERIFY_MDSRV(ap, out_free, rc = RC_PRINT_HELP);
+        if (ap->fs_op != FS_COPY) {
```

Comment:
@mchaarawi I think it can be removed since svc is no longer required to be passed in, but I was doing that because I had separate src and dst svcls and that was only checking one svc